pkg/util/cmd/signal.go

```diff
@@ -19,25 +19,42 @@ package cmd
 import (
 	"os"
 	"os/signal"
+	"syscall"
 )
 
+// This function will call os.Exit(...) in case an error did occure.
+type errorExitFn func()
```

Comment:
Do we really need this as a type? I think just returning a `func()` type is fine.

Reply:
removed it